tests/test_itsdangerous/test_timed.py

```diff
@@ -66,6 +67,18 @@ def test_malformed_timestamp(self, signer):
         assert "Malformed" in str(exc_info.value)
         assert exc_info.value.date_signed is None
 
+    @pytest.mark.skipif(
+        sys.platform == "win32", reason="Freezegun Invalid argument occurs on Windows"
```

Comment:
This shouldn't be ignored. It's not freezegun that fails, it's datetime. Apparently datetime on Windows raises an OSError for an invalid timestamp value. This is expected since the handling is based on the operating system: https://bugs.python.org/issue45031

Reply:
Good catch, I see you fixed it and merged. Thanks!